### ISSUE TITLE
Use NextJS14 with app router instead of pages.

### ISSUE THREAD

**Issue by zairigimad** [api-platform/api-platform#2591]
**Description**  

**Example**  

Hello, i saw the good news of merging the support of nextjs 14, i think it’s missing only the new app router approach https://nextjs.org/docs/app 

i don’t know all the impacts of this change, maybe i can help on this with some guidance of other contributors.

**Comment by dunglas** [comment]
Good idea need to upgrade the code generator first: https://github.com/api-platform/create-client

**Comment by soyuka** [comment]
Closing per #2987 — this repository is becoming a CLI installer. The previous Docker/template distribution is being superseded by the installer + upstream [symfony-docker](https://github.com/dunglas/symfony-docker). If the issue still reproduces with the new setup, please open it on `dunglas/symfony-docker` (for Docker/Caddy/FrankenPHP issues) or `api-platform/core` (for framework issues).
